test/test_corpus.cc

```diff
@@ -152,11 +152,16 @@ TEST_P(ExpectationTest, PerPhaseTest) { // NOLINT
     core::MutableContext ctx(gs, core::Symbols::root());
     // Parser
     vector<core::FileRef> files;
+    constexpr string_view whitelistedTypedNoneTest = "missing_typed_sigil.rb"sv;
     {
         core::UnfreezeFileTable fileTableAccess(gs);
 
         for (auto &sourceFile : test.sourceFiles) {
-            files.emplace_back(gs.enterFile(test.sourceFileContents[test.folder + sourceFile]));
+            auto fref = gs.enterFile(test.sourceFileContents[test.folder + sourceFile]);
+            if (FileOps::getFileName(sourceFile) == whitelistedTypedNoneTest) {
+                fref.data(gs).strictLevel = core::StrictLevel::False;
```

Comment:
`test_corpus` never runs `index` directly, so it doesn't run `decideStrictLevel`. There aren't any mock options objects, so I'm just explicitly adding this behavior for this one weird test.

This test is really intended for LSP, anyway.